pkg/distributor/push.go

```diff
@@ -190,6 +190,30 @@ func handler(
 	})
 }
 
+// Retryable HTTP codes for otlp collector
+var retryableHTTPCodes = map[int]bool{
```

Comment:
a good point, i suggest to do all the noop moved in another PR, this might be difficult to review if we do everything in one.

Reply:
IMO it is OK if we do it here, especially because we are adding another otlp-related method (`toGRPHHTTPStatusOtlp()`) that should **not** be placed in `push.go`. So, why should we place it in `push.go` in this PR in order to move it to another place in another PR?

Reply:
it is to ease the life of reviewers, if you are ok to have all diffs including otlphandler and others in this PR, I am fine to do it.